assets/schema_input.json

```diff
@@ -103,6 +103,20 @@
                 ],
                 "default": "NA",
                 "errorMessage": "The valid input for sample sex is M, F, NA, other, 0, 1 or 2"
+            },
+            "dna_vcf_gz": {
```

Comment:
I think that we can skip the `_gz` ending of this parameter in order to make it a little bit clearer. Good that it takes it though